test/opt/local_ssa_elim_test.cpp

```diff
@@ -2185,13 +2185,13 @@ TEST_F(LocalSSAElimTest, DebugValueForReferenceVariable) {
 ; CHECK: [[f_name:%\w+]] = OpString "f"
 ; CHECK: [[i_name:%\w+]] = OpString "i"
 ; CHECK: [[x_name:%\w+]] = OpString "x"
-; CHECK: [[dbg_f:%\w+]] = OpExtInst %void [[ext:%\d+]] DebugLocalVariable [[f_name]]
+; CHECK: OpExtInst %void [[ext:%\d+]] DebugLocalVariable [[f_name]]
 ; CHECK: [[dbg_i:%\w+]] = OpExtInst %void [[ext]] DebugLocalVariable [[i_name]]
-; CHECK: [[dbg_x:%\w+]] = OpExtInst %void [[ext]] DebugLocalVariable [[x_name]]
+; CHECK: OpExtInst %void [[ext]] DebugLocalVariable [[x_name]]
 
 ; CHECK:      OpStore %f %float_0
-; CHECK-NEXT: OpExtInst %void [[ext]] DebugValue [[dbg_f]] %float_0
-; CHECK-NEXT: OpExtInst %void [[ext]] DebugValue [[dbg_x]] %float_0
+; CHECK-NEXT: OpExtInst %void [[ext]] DebugValue {{%\w+}} %float_0
+; CHECK-NEXT: OpExtInst %void [[ext]] DebugValue {{%\w+}} %float_0
```

Comment:
Why are you not checking the local variables anymore?  If the issue is order, you can use CHECK-DAG.

Reply:
Thanks. CHECK-DAG is what I needed.